css-scroll-snap-2/Overview.bs

```diff
@@ -301,6 +301,25 @@ Snap Events {#snap-events}
 		</tbody>
 	</table>
 
+	<h3 id="event-handlers-on-elements-and-document-objects">Event handlers on elements
+	and Document objects</h3>
+
+	The following are the <a>event handlers</a> (and their corresponding <a>event handler event types</a>)
+	that must be supported by all <a>HTML elements</a>, as both <a>event handler content attributes</a>
+	and <a>event handler IDL attributes</a>:
+
+	<table class="data" dfn-type=attribute dfn-for="Document,Element">
+		<tr>
+			<th><a>Event handler</a></th>
+			<th><a>Event handler event type</a></th>
+		<tr>
+			<td><dfn>onsnapchanged</dfn></td>
+			<td>{{snapchanged}}</td>
+		<tr>
+			<td><dfn>onsnapchanging</dfn></td>
+			<td>{{snapchanging}}</td>
+	</table>
+
```

Comment:
I think these additions to the html spec would belong in an appendix as a modification of the the particular spec section, specifically https://html.spec.whatwg.org/#event-handlers-on-elements,-document-objects,-and-window-objects in this case. E.g. in the scroll-animations spec we have two animation concepts in the appendix labeled as should be moved to the relevant web-animations and css-animations specs: https://drafts.csswg.org/scroll-animations-1/#timeline-ranges

Reply:
Thanks for the link. Done.